src/Repositories/SearchIndexRepository.php

```diff
@@ -217,56 +214,43 @@ protected function getPrefixedTerms($term)
      *
      * @return void
      */
-    public function createSearchIndexes($brand)
+    public function createSearchIndexes(): void
```

Comment:
Is there a way to force a rebuild of the entire table from a command?  Would be nice to have that option in case we need to.

Reply:
Good idea. Consider it done